.github/workflows/rill-cloud.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow triggers deployment with Rill Cloud helm charts
+# Each merge to main branch is build with github sha tag and published to Rill Cloud.
+name: Deploy to Rill Cloud
+on:
+  push:
+    tags:
+      - "**"
+    branches:
+      - main
+      - cd
+  workflow_dispatch:
+
+env:
+  RELEASE: ${{ contains('refs/tags', github.ref_name) }}
+
+jobs:
+  release:
+    name: Deploy to Rill Cloud
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.2
+
+      - name: Set up GCloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Authenticate GCloud
+        uses: google-github-actions/auth@v1
+        with:
+          credentials_json: "${{ secrets.RILL_BINARY_SA }}"
+
+      - name: Build & Publish Rill docker image
+        run: |-
+          go build -o rill cli/main.go
```

Comment:
For goreleaser, we actually pass a few extra options:

```
go build -o rill -mod=readonly -ldflags="-s -w -X main.Version={{.Version}} -X main.Commit=${{.ShortCommit}} -X main.BuildDate={{.Date}}" cli/main.go
```

They are not important, but wondering if we should set some of them here also (could maybe be useful for debugging? We could expose them on a `/meta` endpoint or something). "Version" won't really make sense, maybe we just set it to `"main"` or `"headless"`?

Reply:
version can be SHA?

Reply:
Yeah could be, although that would also be in `main.Commit`. Maybe it would be useful to have a way of knowing that it's a headless build?